README.md

```diff
@@ -56,6 +56,12 @@ rescue Measured::UnitError
 end
 ```
 
+Parse from string without having to split out the value and unit first:
+
+```ruby
+Measured::Weight.parse("123 grams")
```

Comment:
nit: to be aligned with the other comments, show the `#inspect` after. Maybe also show that you don't need a space.